### ISSUE TITLE
Documentation out of date (Ubuntu 16.04 installation of CMS, third step ):

### ISSUE THREAD

**Issue by romeorizzi** [cms-dev/cms#896]
This is where I get the problem:

```
romeo@romeo-HP-ProBook-450-G4:~/Dropbox/corsi/Algoritmi/esercitazioniCMSalgo/2018$ sudo apt-get install nginx-full php7.0-cli php5-fpm phppgadmin \
>      texlive-latex-base a2ps gcj-jdk haskell-platform
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package php5-fpm is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'php5-fpm' has no installation candidate
romeo@romeo-HP-ProBook-450-G4:~/Dropbox/corsi/Algoritmi/esercitazioniCMSalgo/2018$ 
```

**Comment by lerks** [comment]
Thanks for the report! Apparently this needs to be upgraded to `php7.0-fpm` in Ubuntu 16.04 (for cms v1.3) and to `php7.2-fpm` in Ubuntu 18.04 (for cms master).

**Comment by wil93** [comment]
Wasn't this already updated here? 1cdd53319613ec7d2ce01162c4029563a35508f9

**Comment by lerks** [comment]
Thanks for digging that up! It does indeed fix it, but it was never backported to v1.3. (Also, for master we now need to change it to php7.2). I will do that.

**Comment by lerks** [comment]
Fixed. Changes pushed as 4e4dc4370b2b67ad090debe36907b7b557dfac99 (v1.3) and 5669a7bddef4dd09cb0732fd6795df069145d093 (master).
